packages/valory/skills/safe_deployment_abci/behaviours.py

```diff
@@ -59,23 +60,23 @@ def synchronized_data(self) -> SynchronizedData:
 class RandomnessSafeBehaviour(RandomnessBehaviour):
     """Retrieve randomness for oracle deployment."""
 
-    behaviour_id = "randomness_safe"
+    behaviour_id = States.RANDOMNESS_SAFE.value
```

Comment:
Alternatively, we can just autogenerate these framework side. Might be simpler still and reduce LOC.